importers/nmdb-import-cisco/Parser.cpp

```diff
@@ -450,6 +660,26 @@ Parser::getData()
     }
   }
 
+  // Apply interface apply-groups to affected rules
+  for (auto& [bookName, dataPair] : usedRuleBooks) {
+    auto& [ifaceName, direction] = dataPair;
+
+    for (auto& [id, rule] : d.ruleBooks[bookName]) {
+      if ("in" == direction) {
+        // in: filter traffic entering iface, regardless destination
+        rule.addSrcIface(ifaceName);
+        rule.addDstIface("any");
+      } else if ("out" == direction) {
+        // out: filter traffic leaving iface, regardless origin
+        rule.addSrcIface("any");
+        rule.addDstIface(ifaceName);
+      } else {
+        LOG_ERROR << "Parser::assignRules: Unknown rule direction parsed: "
+                  << direction << std::endl;
+      }
+    }
+  }
+
```

Comment:
We only have `global` zones in this file currently, and in the foreseeable future. Do we still want to consider adding the state explosion calls here for `NetBooks`, `ServiceBooks`, and `RuleBooks` just in case? As far as I know, the rest of the logic in this parser should remain the same, minus adding logic to change the zone name.